dashboard/app/controllers/admin_users_controller.rb

```diff
@@ -104,25 +117,35 @@ def manual_pass
   # get /admin/permissions
   def permissions_form
     search_term = params[:search_term]
-    if search_term =~ /^\d+$/
-      @user = User.find(search_term)
-    elsif search_term.present?
-      users = User.where(hashed_email: User.hash_email(search_term))
-      @user = users.first
-      if users.count > 1
-        flash[:notice] = "More than one User matches email address.  "\
-                         "Showing first result.  Matching User IDs - #{users.map(&:id).join ','}"
+    permission = params[:permission]
+    if search_term.present?
+      if search_term =~ /^\d+$/
+        @user = User.select(USER_ATTRIBUTES_FOR_VIEW).find_by(id: search_term)
+      else
+        users = User.where(hashed_email: User.hash_email(search_term))
+        @user = users.select(USER_ATTRIBUTES_FOR_VIEW).first
```

Comment:
Is it possible to invoke an ActiveRecord QueryMethod (`select`) on `@user` after these 2 branches complete execution?

The change below raises the error `private method 'select' called for #<User:0x007f84dbdc7e10>`:

``` ruby
      if search_term =~ /^\d+$/
        @user = User.find_by(id: search_term)
      else
        users = User.where(hashed_email: User.hash_email(search_term))
        @user = users.first
        if users.count > 1
          flash[:notice] = "More than one User matches email address.  "\
                         "Showing first result.  Matching User IDs - #{users.pluck(:id).join ','}"
        end
      end
      @user = @user.select(USER_ATTRIBUTES_FOR_VIEW) if @user
```